### ISSUE TITLE
Unit tests fail with PHP 7.1

### ISSUE THREAD

**Issue by duffn** [wp-graphql/wp-graphql#205]
The `test-theme-connection-queries` and `test-plugin-queries` tests fail when run with PHP 7.1.

```
% php --version
PHP 7.1.8 (cli) (built: Aug  7 2017 15:02:19) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.1.0, Copyright (c) 1998-2017 Zend Technologies

% phpunit --version
PHPUnit 6.3.0 by Sebastian Bergmann and contributors.
```

```
% phpunit

There were 2 failures:

1) WP_GraphQL_Test_Plugin_Queries::testPluginsQuery
Failed asserting that a NULL is not empty.

/Users/nicholasduffy/code/wp-graphql/tests/test-plugin-queries.php:72

2) WP_GraphQL_Test_Theme_Queries::testThemesQuery
Failed asserting that a NULL is not empty.

/Users/nicholasduffy/code/wp-graphql/tests/test-theme-connection-queries.php:65
```
Both of the `do_graphql_request`s output:
```
Array
(
    [data] => Array
        (
            [themes] =>
        )

    [errors] => Array
        (
            [0] => Array
                (
                    [message] => A non-numeric value encountered
                    [locations] => Array
                        (
                            [0] => Array
                                (
                                    [line] => 3
                                    [column] => 5
                                )

                        )

                    [path] => Array
                        (
                            [0] => themes
                        )

                )

        )

)
```

These tests pass with
```
% php --version
PHP 7.0.22 (cli) (built: Aug  7 2017 14:06:15) ( NTS )
Copyright (c) 1997-2017 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2017 Zend Technologies
```

**Comment by jasonbahl** [comment]
@duffn thanks for the detailed info!

**Comment by jasonbahl** [comment]
addressed in #211
